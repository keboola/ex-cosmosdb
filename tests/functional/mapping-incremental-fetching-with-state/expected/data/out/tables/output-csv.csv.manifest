{
    "incremental": false,
    "columns": [
        "id",
        "full_name",
        "age"
    ],
    "column_metadata": {
        "id": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "string"
            }
        ],
        "full_name": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "string"
            }
        ],
        "age": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "string"
            }
        ]
    }
}