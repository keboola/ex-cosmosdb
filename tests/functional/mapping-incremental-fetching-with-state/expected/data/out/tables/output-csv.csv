"4","Zane Craft","13"
"5","Sebastien Mora","32"
